### PR TITLE
Lo2152 multicall

### PR DESCRIPTION
**Pull request by dai1975**
target v0.3.7(multicall merge)
add multicall test

### CODE REVIEW

File: tests/chains/ethereum/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ docker-images:
 
 .PHONY: deploy-contract
 deploy-contract:
-	$(DOCKER_COMPOSE) run contracts sh -c 'npm install && npx hardhat run ./scripts/deploy.js --network ibc0 && npx hardhat run ./scripts/deploy.js --network ibc1'
+	$(DOCKER_COMPOSE) run contracts sh -c 'npm install &&  npx hardhat run ./scripts/deploy.js --network ibc0 && npx hardhat run ./scripts/deploy.js --network ibc1'
```
Comment (siburu) [inline tests/chains/ethereum/Makefile]: ?

**Review by siburu — COMMENTED** [review]
@dai1975 Almost perfect, but I left just one comment.

File: tests/chains/ethereum/Makefile (hunk 1 above)
Comment (dai1975) [reply tests/chains/ethereum/Makefile]: It may miss typing. fixed.

**Review by siburu — APPROVED** [review]
@dai1975 LGTM. Thank you!

**Pull request merged**